data-prepper-plugins/s3-source/src/main/java/com/amazon/dataprepper/plugins/source/BackoffUtils.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.source;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Random;
+
+public class BackoffUtils {
```

Comment:
In place of utils could `BackoffUtils` class have a more descriptive name? Consider something like `RetryState` or `BackoffManager`. Without looking at the class file I expected BackoffUtils to contain some helper functions for parsing or transforming data.